File: Service_eHospital_DongNai_A/Implement/ztmp_BCQR_01Service.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class ztmp_BCQR_01Service : BaseService<ztmp_BCQR_01, Iztmp_BCQR_01Repository>
    , Iztmp_BCQR_01Service
    {
    private readonly Iztmp_BCQR_01Repository _ztmp_BCQR_01Repository;
    public ztmp_BCQR_01Service(Iztmp_BCQR_01Repository ztmp_BCQR_01Repository) : base(ztmp_BCQR_01Repository)
    {
    _ztmp_BCQR_01Repository = ztmp_BCQR_01Repository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/ztmp_BCQR_01.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ztmp_BCQR_01 } from './ztmp_BCQR_01.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ztmp_BCQR_01Service extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'sTT', 'soHoaDon', 'soTaiKhoan', 'soHoaDonQR', 'giaTriHoaDon', 'thoiGianHoaDon', 'LoaiThu', 'soVaoVien', 'maDotDieuTri', 'tenBenhNhan', 'namSinh', 'soThamChieu', 'maGiaoDich', 'nguoiThu', 'dienGiai'];

    List: ztmp_BCQR_01[] | undefined;
    ListFilter: ztmp_BCQR_01[] | undefined;
    FormData!: ztmp_BCQR_01;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ztmp_BCQR_01";
    }
}

